dev-aws/kafka-shared-msk/pubsub/pubsub-examples.tf

```diff
@@ -0,0 +1,33 @@
+resource "kafka_topic" "pubsub_examples" {
```

Comment:
Do you need this? We don't have it for `kafka-shared`, is there anything different about this example or was it for testing purposes?